src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -398,10 +420,37 @@ void GraphOptimizer::FuseFCAndWeightsDecompression(Graph &graph) {
                 continue;
         }
 
+        size_t IC = fcInputWeightsShape.getDims()[1];
+        if (IC % groupNum != 0 || IC / groupNum < 4) {
```

Comment:
Could you please elaborate why we don't fuse decompression if `IC / groupNum < 4`? Is it oneDNN limitation or we just don't want to fuse it because we will not get performance benefit from decompression fusion in this corner case?

Reply:
Yes, This is impl limitations. Added corresponding comment